CheatSheets/php-cheatsheet.md

```diff
@@ -1,29 +1,123 @@
-| `\"` | Print the next character as a double quote, not a string closer
+| `// Comment Message` | This is a single-line comment |
+| `$Title = "Title"` | Defining Variables|
+| `\n` | New Line|
+| `\r` | It inserts a carriage return in the text at this point.|
+| `\t` | It gives a horizontal tab space|
+| `\v` | It gives a vertical tab space|
+| `\\` | It adds a backslash|
+| `\$` | Print the next character as a dollar, not as part of a variable|
+| `\'` | Print the next character as a double quote, not a string closer|
+| `\"` | Print the next character as a double quote, not a string closer|
+
+**[🔼Back to Top](#table-of-contents)**
+
+## Operators
+
+|operator|name |syntax|type|
+|---------|------|-----|----|
+|`+`|	Addition|	$x + $y|Arithmetic Operators|
+|`-`|	Subtraction|	$x - $y|Arithmetic Operators|
+|`*` |	Multiplication|	$x * $y |Arithmetic Operators|
+| `/` |	Division|	$x / $y|Arithmetic Operators|	
+|`%`|	Modulus|	$x % $y|	Arithmetic Operators|y	
+|` ** `|	Exponentiation |	$x ** $y	|Arithmetic Operators|	
+|`==`|	Equal|	$x == $y|Comparison Operators|	
+|`===`|	Identical|	$x === $y|Comparison Operators|
+| `!= `| 	Not equal|	$x != $y|Comparison Operators|	
+| `<>` |	Not equal |	$x <> $y	|Comparison Operators|	
+|`!==` |	Not identical|	$x !== $y |Comparison Operators|	
+|`>`|	Greater than|	$x > $y|Comparison Operators|	
+|`<`|	Less than|	$x < $y|Comparison Operators|	
+|`>=`|	Greater than or equal to|	$x >= $y|Comparison Operators|	
+|`<=`|	Less than or equal to|	$x <= $y|Comparison Operators|	
+|`<=>`|	Spaceship|	$x <=> $y	|Comparison Operators|
+|`and`|	And|	$x and $y|Logical	Operators|
+|`or`	|Or|	$x or $y	|Logical	Operators|
+|`xor`|	Xor|	$x xor $y	|Logical	Operators|
+|`&&`|	And|	$x && $y|	Logical	Operators|
+| ![image](https://user-images.githubusercontent.com/47249568/197706502-62d506bd-5596-4c5f-a1e3-7e58633c4139.png)	|Or|	$x ![image](https://user-images.githubusercontent.com/47249568/197706528-2bbe3369-5c51-4e1b-b295-c607f05bfa77.png) $y |Logical	Operators|	
+|`!`	|Not	|!$x |	Logical	Operators|
+|`.`|	Concatenation|	$txt1 . $txt2	|String Operators|
+|`.=`|	Concatenation assignment|	$txt1 .= $txt2|	String Operators|
+|`+`|	Union|	$x + $y|Array Operators|	
+|`==`|	Equality|	$x == $y	|Array Operators|
+|`===`|	Identity|	$x === $y |Array Operators|
+|`!=`|	Inequality|	$x != $y	|Array Operators|
+|`<>`|	Inequality|	$x <> $y	|Array Operators|
+|`!==`|	Non-identity|	$x !== $y	|Array Operators|	
+|`?:	`|Ternary|	$x = expr1 ? expr2 : expr3|	Conditional Assignment Operators|
+|`??`|	Null coalescing|	$x = expr1 ?? expr2|	Conditional Assignment Operators|
+
+
+|operator|name |description|type|
+|---------|------|-----|----|
+|`++$x`|	Pre-increment|	Increments $x by one |Increment|
+|`$x++`|	Post-increment|	Returns $x, then increments $x by one	|Increment|
+|`--$x`|	Pre-decrement	|Decrements $x by one, then returns $x|decrement|	
+|`$x--`	|Post-decrement	|Returns $x, then decrements $x by one	|decrement|	
```

Comment:
Could you divide all the above operators like `Increment & Decrement` Operators? 
For example, `Arithmetic`, `Comparison`, `Logical`, `String`, `Array`, `Conditional` Operations etc.